extmod/asyncio/stream.py

```diff
@@ -156,14 +171,23 @@ async def _serve(self, s, cb):
             except:
                 # Ignore a failed accept
                 continue
+
             s2.setblocking(False)
+            if ssl:
+                try:
+                    s2 = ssl.wrap_socket(s2, server_side=True, do_handshake_on_connect=False)
+                    s2.setblocking(False)
+                except OSError as e:
+                    sys.print_exception(e)
```

Comment:
Instead of importing sys, use `core.sys.print_exception(e)`.  It will be slightly smaller code and use slightly less RAM (in the globals dict of this module).

Reply:
Also you can probably move the `s2.setblocking(False)` to after this `if ssl:` block, and remove line 175 above.